js/script.js

```diff
@@ -0,0 +1,298 @@
+        img.src = element.srcPics;
+        img.alt = 'picture';
+        hangmanWrapper.appendChild(img);
+    });
+
+}
+createHangmanPics(arrayHangmanBody);
+
+// Add title Hangman
+const hangmanTitle = document.createElement('h4');
+hangmanTitle.classList.add('hangman_title');
+hangmanTitle.textContent = 'Hangman Game';  //Hangman  game title
+hangmanGame.appendChild(hangmanTitle);
+
+// Create ul-List
+const createList = () => {
+    const listWords = document.createElement('ul');
+    listWords.classList.add('hangmanGame__list');  //List Words
+    hangmanKeyboards.appendChild(listWords);
+
+    return listWords;
+}
+const listWords = createList();
+
+// Create li-List Item
+const createListItem = (list, length) => {
+
+    for (let i = 1; i <= length; i++) {
+
+        const listItem = document.createElement('li');
+        listItem.classList.add('hangmanGame__list_item');  //List item
+        list.appendChild(listItem);
+    }
+}
+
+// Create question block
+const createElemQuestion = () => {
+    const question = document.createElement('p');
+    question.classList.add('question_game');
+    hangmanKeyboards.appendChild(question);
+
+    return question;
+}
+const returnQuestion = createElemQuestion();
+
+// Create attempts block
+const createAttempts = () => {
+    const attempts = document.createElement('p');
+    attempts.classList.add('attempts_games');
+    attempts.innerHTML = `Incorrect guesses: <span>0 / 6</span>`; //Attempts block
+    hangmanKeyboards.appendChild(attempts);
+
+    return attempts;
+}
+const returnAttempt = createAttempts();
+
+// Create keyboard container
+const createKeyboardContainer = () => {
+    const keyboardsContainer = document.createElement('div');
+    keyboardsContainer.classList.add('keyboards__container'); //Keyboards Container
+    hangmanKeyboards.appendChild(keyboardsContainer);
+
+    return keyboardsContainer;
+}
+const keyboardContainer = createKeyboardContainer();
+
+// Create keyboard buttons
+const createKeyboard = (container) => {
+    for (let i = 65; i <= 90; i++) {
+        const keyboardsBtn = document.createElement('button');
+        keyboardsBtn.type = 'button';
+        keyboardsBtn.classList.add('keyboards__button');
+        keyboardsBtn.setAttribute('data-code', `Key${String.fromCharCode(i)}`);
+        keyboardsBtn.innerText = String.fromCharCode(i); // Keyboard
+        container.appendChild(keyboardsBtn);
+    }
+}
+createKeyboard(keyboardContainer);
+
+// KeyA => a
+const formaKeyboardCode = (code) => code[code.length - 1].toLowerCase();
+
+
+// Get Random questions
+const getRandomQuestion = (question) => {
+    const { word, hint } = wordList[Math.floor(Math.random() * wordList.length)];
+    console.log('correct answer: ', word);
+    question.innerHTML = 'Hint: ' + `<b>${hint}</b>`;
+    createListItem(listWords, word.length);
+
+    return word;
+}
+const answer = getRandomQuestion(returnQuestion);
+
+//Open modal window
+const callPopUpWindow = () => {
```

Comment:
```suggestion
const checkIfAllItemsFilled = () => {
```
функция не вызывает модалку, она проверяет, что все лишки заполнены